documentation/library_guide/introduction.rst

```diff
@@ -177,5 +177,7 @@ Known Limitations
   than ``2423.32`` (Rolling) and ``2350.61`` (LTS) is used.
   See the `Release Types <https://dgpu-docs.intel.com/releases/releases.html>`_
   to find information about the relevant Rolling and LTS releases.
+* ``sort``, ``stable_sort``, ``sort_by_key`` and ``stable_sort_by_key`` algorithms with a device policy are still not supported
+  on Windows with Intel® Arc™ graphics 140V.
```

Comment:
Is this a temporary limitation or a "permanent" one? If we expect this to be resolved in reasonable time, the note should go into Release Notes, not into this document.

Reply:
It has not been investigated yet. I will take a look at it. I hope to fix it by the next release, so let's not to add it here.